### DISCUSSION TITLE
Packaging failed staus UNKNOWN

### DISCUSSION THREAD

**Discussion by ThomasDV1984** [opengisch/QField#5065] [Bugs]
We are unable to package a previously working project. The job llog gives the warning 
"06:44:32.156 QGSSTDERR WARNING QFSFileEngine::open: No file name specified
06:44:32.156 root INFO Stopping QGIS app…"
![image](https://github.com/opengisch/QField/assets/77888654/423dd620-89a4-4ae9-99c2-8210dfdaa536)

The problem started when we changed the setting "Packaging Offliner" to Optimized Packager"

![image](https://github.com/opengisch/QField/assets/77888654/65af32af-4ac4-498d-bcf4-a267c1ff801e)

Setting it back to "Qgis Core Offline Editing" pack-ging works but the objects of the pointlayer don't get pushed with the package. Other objects in other layers are pushed to the mobile project.

**Comment by SeqLaz** [comment]
@ThomasDV1984  could you try synchronizing your project using the new Optimized packager?. To do so, please navigate to the Settings of your project and make the changes to the packager.

You can find more information about this optimized packager at: https://docs.qfield.org/get-started/tutorials/get-started-qfc/#enhance-your-project-with-the-optimized-packager

**Reply by ThomasDV1984** [reply]
The problem started when we changed it to 'Optimized packager'

**Reply by SeqLaz** [reply]
It appears that certain features are attempting to incorporate geometry into layers that do not support it. You could try retrieving your data from the mobile device by:

1. Navigate to Settings > Open Project Folder > data.gpkg on your mobile device.
2. Utilize the three dots on the right side and select the "Send to..." function to transfer your data to your computer.
3. Inspect those features directly in QGIS for further analysis.

**Reply by BoswachterMarc** [reply]
@ThomasDV1984 The error message says 'Geometry type is not compatible with current layer'. Check if it is not related to https://github.com/opengisch/qfieldcloud/issues/738
That bug with linestringZ still exists.

**Reply by ThomasDV1984** [reply]
It does still exist. We had an other project that I had to get of the server and then put it back online and then it worked again. I don(t know where the z values come from, because I make sure when creating a new project all layers are without the z value.

**Comment by BoswachterMarc** [comment]
> @ThomasDV1984 The error message says 'Geometry type is not compatible with current layer'. Check if it is not related to [opengisch/qfieldcloud#738](https://github.com/opengisch/qfieldcloud/issues/738) That bug with linestringZ still exists.

You can check your line layer(s) for z-values with the processing toolbox and 'extract z values'. If there are z-values, then that is the problem. Delete the feature with z-values or replace them, and the problem is probably solved. 
![image](https://github.com/opengisch/QField/assets/57868975/cd5837d4-5cbb-44a1-9234-38c4e2276e23)
